### PR TITLE
Digital Ocean API v2 redux

### PR DESCRIPTION
**Pull request by Afterglow**
This builds upon #139 by removing the leftover pry line and updating the documentation to reflect how to get this to work per my discussion with @jjasghar.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,15 +19,22 @@ These are the primary documents to help learn about using Provisioning and creat
 
 ## chef-provisioning-fog Usage and Examples
 
+**A note about key pairs** - The key name used in `fog_key_pair` must be the same as the filename of the local key to be used. If the key does not exist in one of `private_key_paths` (which you can set in knife.rb or in a client.rb) it will be created.
+
 ### DigitalOcean
 
+<<<<<<< HEAD
```
Comment (jjasghar) [inline README.md]: Probably want to remove this? :)

File: README.md [hunk 2]
```diff
@@ -19,15 +19,22 @@ These are the primary documents to help learn about using Provisioning and creat
...
+Update your knife.rb to contain your DigitalOcean API token and the driver
+=======
 If you are on DigitalOcean and using the [`tugboat` gem](https://github.com/pearkes/tugboat), you can do this:
+>>>>>>> upstream/master
```
Comment (jjasghar) [inline README.md]: Probably want to remove this? :)

**Comment by jjasghar** [comment]
After this is merged, this #147 needs to be cleaned up too.

**Comment by Afterglow** [comment]
Hrm sorry I was sure I fixed that. Gone now.

**Comment by jjasghar** [comment]
I completely missed @martinb3 's #130 which references the issues we were talking about yesterday. With this documentation i think we can close that issue now too.

**Pull request merged**